File: DentOffice.Model/Pacijent_Data.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DentOffice.Model
{
    public class Pacijent_Data
    {
        public int PacijentID { get; set; }
        public bool Proteza { get; set; }
        public bool Aparatic { get; set; }
        public bool Terapija { get; set; }
        public int KorisnikID { get; set; }

    }
}

File: dentofficemobile/lib/models/pacijent_data.dart
// -- pacijent_data.dart --
import 'package:json_annotation/json_annotation.dart';

part 'pacijent_data.g.dart';

@JsonSerializable()
class Pacijent_Data {
  int pacijentID;
  bool proteza;
  bool aparatic;
  bool terapija;
  int korisnikID;

  Pacijent_Data({required this.pacijentID,required this.proteza,required this.aparatic,required this.terapija,required this.korisnikID,});

  factory Pacijent_Data.fromJson(Map<String, dynamic> json) => _$Pacijent_DataFromJson(json);

  Map<String, dynamic> toJson() => _$Pacijent_DataToJson(this);
}